test/integration/verify/controls/azurerm_management_groups.rb

```diff
@@ -0,0 +1,18 @@
+tenant_id = attribute('tenant_id', default: nil)
+
+control 'azurerm_management_groups' do
+  describe azurerm_management_groups do
+    its('group_ids')           { should include "/providers/Microsoft.Management/managementGroups/#{tenant_id}" }
```

Comment:
Testing this one I see:
```
     ×  Management Groups group_ids should include "/providers/Microsoft.Management/managementGroups/<tenant_id>"
     expected ["/providers/Microsoft.Management/managementGroups/mg_child_two", "/providers/Microsoft.Management/managementGroups/mg_child_one", "/providers/Microsoft.Management/managementGroups/mg_parent"] to include "/providers/Microsoft.Management/managementGroups/<tenant_id>"
```
I would suggest either to remove this line or update it to:
```
"/providers/Microsoft.Management/managementGroups/mg_parent"
```

Reply:
I think these may be failing for you because the service principal you're using may not have the correct permissions. There are a couple of things that must be done to grant proper perms.

1. You need to grant yourself "Access management for Azure resources", read more [here](https://docs.microsoft.com/en-us/azure/role-based-access-control/elevate-access-global-admin).
2. You need to grant "Contributor" to the service principal on the built-in root group.

Reply:
In any case I can technically remove these failing tests to make it simpler for future devs, but I could also just update the documentation on how these permissions should be granted. What do you think is best?

Reply:
To simplify execution of the tests I'd suggest removing those requiring special permissions.  Including the above instructions in the resource README would be a great addition, many thanks!